apps/site/components/withChangelogModal.tsx

```diff
@@ -62,7 +62,7 @@ const WithChangelogModal: FC<WithChangelogModalProps> = ({
             }
           );
         }
-      } catch (_) {
+      } catch {
```

Comment:
Behavior change in ESLint v9 and typescript-eslint v8: catch blocks are now also checked by default in [`no-unused-vars`](https://eslint.org/docs/latest/rules/no-unused-vars`) -> [`@typescript-eslint/no-unused-vars`](https://typescript-eslint.io/rules/no-unused-vars). 🧹